app/controllers/user/team_players_controller.rb

```diff
@@ -11,6 +11,23 @@ def destroy
   end
 
   def create
+    player = Player.find_by(display_name: params[:myPlayer])
+    team = current_user.teams.find(params[:team_id])
+    if player
+      team_player = TeamPlayer.find_or_initialize_by(player: player, team: team)
+      if team_player.new_record?
+        team_player.save
+        flash[:success] = "Player added!"
+      else
+        flash[:error] = "Player already on team."
+      end
+    else
+      flash[:error] = "No player found matching that name."
+    end
+    redirect_to user_team_path(team)
+  end
+
+  def create_from_player_search
```

Comment:
Thanks for re-naming this

Reply:
The plan is to get these 2 merged into one create method, but I just renamed for now so we can all work out of the same code base.